orc8r/tools/ansible/roles/python_dev/tasks/main.yml

```diff
@@ -51,18 +51,14 @@
 # Add common convenience aliases
 #################################
 
-- name: Set a convenience function for activating the virtualenv
+- name: Set functions for virtualenv
```

Comment:
Nit: I would say it's easier to understand what's going on here at a glance with "convenience functions" in the name rather than just "functions"